app/views/hyrax/dashboard/collections/_show_add_items_actions.html.erb

```diff
@@ -1,16 +1,21 @@
 <h2 class="sr-only"><%= t('hyrax.collection.actions.header') %></h2>
 <div class="actions-controls-collections">
   <% if can? :edit, presenter.solr_document # TODO: Shouldn't this be :deposit -- dependency on participants %>
-    <%= link_to t('hyrax.collection.actions.add_works.label'),
+    <%= link_to t('hyrax.collection.actions.add_existing_works.label'),
                 hyrax.my_works_path(add_works_to_collection: presenter.id),
-                title: t('hyrax.collection.actions.add_works.desc'),
+                title: t('hyrax.collection.actions.add_existing_works.desc'),
                 class: 'btn btn-default pull-right',
                 data: { turbolinks: false } %>
+    <%= link_to t('hyrax.collection.actions.add_new_work.label'),
+                '#',
+                title: t('hyrax.collection.actions.add_new_work.desc'),
+                data: { behavior: "select-work", target: "#worktypes-to-create", 'create-type' => 'single', add_works_to_collection: presenter.id },
+                class: 'btn btn-default  pull-right'  %>
     <% if presenter.collection_type_is_nestable? %>
-        <%= link_to t('hyrax.collection.actions.nest_collections.label'),
-                    hyrax.dashboard_new_nest_collection_within_path(child_id: presenter.id),
-                    title: t('hyrax.collection.actions.nest_collections.desc'),
-                    class: 'btn btn-default' %>
+    <%= link_to t('hyrax.collection.actions.nest_collections.label'),
```

Comment:
Don't we want this indented so it's clear this is within the `if` block?

Reply:
There are some places in the code that seem to try and make the produces html indent correctly (which is what is seen here) and others that make the erb indent correctly (like in _form above).  I'm not sure which is 'better', so I'll go with your suggestion.  It's pretty inconsistent in the code.